package/etc/conf.d/log_paths/lp-mcafee_nsp.conf.tmpl

```diff
@@ -0,0 +1,71 @@
+# McAfee NSP
+
+{{- /* The following provides a unique port source configuration if env var(s) are set */}}
+{{- $context := dict "port_id" "McAfee_NSP" "parser" "rfc5424_noversion" }}
+{{- tmpl.Exec "t/source_network.t" $context }}
+
+filter f_mcafee_nsp{
+  match("mcafee_nsp", value("fields.sc4s_vendor_product") type(glob));
```

Comment:
because this message is valid BSD, and there are 3 unique values of PROGRAM we should use that as the filter so this can work with zero config oob.